settings.gradle

```diff
@@ -1 +1,17 @@
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        google()
+        mavenCentral()
+    }
+}
+
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
+        mavenCentral()
+    }
+}
+
 include ':parsely', ':example'
```

Comment:
Suggestion (💡): You might also consider splitting the `include` into separate lines so that when and if you do an update to any of the modules the change for a specific module will apply to a specific line thus not affecting any other module (just to be on the safe side). Also, with this pattern you don't need to worry about commas (`,`) separating the modules:
```
include ':parsely'
include ':example'
```

Reply:
Thanks! I prefer having a single `include` but I've moved declarations into separate lines.